src/NUnitFramework/Directory.Build.props

```diff
@@ -7,7 +7,7 @@
     <AssemblyOriginatorKeyFile>..\..\nunit.snk</AssemblyOriginatorKeyFile>
     <DisableImplicitNuGetFallbackFolder>true</DisableImplicitNuGetFallbackFolder>
     <NUnitLibraryFrameworks>net462;netstandard2.0</NUnitLibraryFrameworks>
-    <NUnitRuntimeFrameworks>net462;netcoreapp3.1;net5.0;net6.0;net7.0</NUnitRuntimeFrameworks>
+    <NUnitRuntimeFrameworks>net462;netcoreapp3.1;net6.0;net7.0</NUnitRuntimeFrameworks>
```

Comment:
Net 5 is STS, whereas netcore 3.1 is LTS.  I would like to get rid of 3.1 too, but a bit hesitant if it is a bit early.  But, since this is nunit 4, we could just do that.  I will simplify more. 
@stevenaw , @jnm2  Comments ?

Reply:
The closest discussion I've found is this:  https://github.com/nunit/nunit/issues/3070
NUnit has had a tradition of long backwards supports, but we also need to consider the high pace of framework and language changes, so removing net core 3.1 could be wise.

Reply:
I've added this discussion #4424 for NUnit lifecycle policy.